reference/cd.md

```diff
@@ -16,6 +16,7 @@ a subset of these code changes after specific gates have been passed (certain te
 * [AWS Code Services](./cd-aws-code-services.html)
 * [Azure DevOps](./cd-azure-devops.html)
 * [CircleCI](./cd-circleci.html)
+* [Codefresh](./cd-codefresh.html)
```

Comment:
Thanks for doing this! 👍

Reply:
I addressed your comments. Let me know if you need anything else :-)